test/klippy/printers.test

```diff
@@ -107,6 +107,7 @@ CONFIG ../../config/generic-duet2-maestro.cfg
 DICTIONARY sam4e8e.dict
 CONFIG ../../config/generic-duet2.cfg
 CONFIG ../../config/generic-duet2-duex.cfg
+CONFIG ../../config/printer-modix-big60-2020.cfg
```

Comment:
Can you please check, why this is marked as changed?

Reply:
sorry, i'm not the best with using GIT
i downloaded the recent release and edited that as my config is most likely old.